### PR TITLE
Article Table and Resolver

### PR DESCRIPTION
**Pull request by Reneehsu**
Create article table and resolver.

### CODE REVIEW

File: src/resolver/article.resolver.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        @Arg('id') id: string,
+    ): Promise<Article | null> {
+        const tbl = await this.table();
+        return await tbl.getArticle(id);
+    }
+
+    @Query(() => [Article], { nullable: true })
+    public async articles(
+        @Arg('ids', () => [String]) ids: string[],
+    ): Promise<Article[] | null> {
```
Comment (jingwanhsu) [inline src/resolver/article.resolver.ts:25]: Can we have a function to get all articles without providing ids?

Comment (Reneehsu) [reply src/resolver/article.resolver.ts:25]: yes! the getAllArticles can do that

Comment (jingwanhsu) [reply src/resolver/article.resolver.ts:25]: oh great! did not see that

File: src/resolver/article-table.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { MongoDBTableBase } from "../mongodb/mongodb-manager";
+import { Article } from "../../common/models";
+import * as _ from "lodash";
+
+export class ArticleTable extends MongoDBTableBase("articles") {
+    public addArticle(article: Article) {
```
Comment (jingwanhsu) [inline src/resolver/article-table.ts:6]: Some comments to the model: 
author -> authors
status -> can it be a enum? or maybe a boolean called isPublished, whichever
pusblishTime -> better be stored as integer. unix timestamp

Some fields I would like:
  slug: string
  preview: string (or excerpt or description or any other name)
Also create time and last modified time should be still useful. both should be in unix timestamp

Comment (jingwanhsu) [reply src/resolver/article-table.ts:6]: just see the type of status is ArticleStatus. Please use that type for graphql as well!

**Pull request merged**